README.md

```diff
@@ -134,6 +134,32 @@ conctr.sendData(currentTempAndPressure, function(error, response) {
 }.bindenv(this));
 ```
 
+### send(*unusedKey, payload[, callback]*)
+
+Alias for *sendData* method above, allows for *conctr.send* to work using the same arguments as the Electic Imp internal `agent.send(key, payload)` 
+
+
+| Key | Data Type | Required | Description |
+| --- | --------- | -------- | ----------- |
+| *unusedKey* | String | Yes | A string that will be ignored. |
+| *payload* | Table | Yes | A table containing the data to be sent to Conctr. This keys in the table should correspond to fields from the model and the keys should be of type specified in the model |
+| *callback* | Function | No | Function to be called on response from Conctr. The function should take two arguements, *error* and *response*. When no error occurred, the first arguement will be null |
+
+#### Example
+
+```squirrel
+local currentTempAndPressure = { "temperature" : 29, "pressure" : 1032};
+
+conctr.send("any string", currentTempAndPressure, function(error, response) {
```

Comment:
Take the callback out of this example but leave it in otherwise.
agent.send() doesn't support callbacks so it will be confusing.